blob/src/main/java/io/crate/http/netty/HttpBlobHandler.java

```diff
@@ -258,7 +258,7 @@ public void exceptionCaught(ChannelHandlerContext ctx, ExceptionEvent e)
             body = ex.getMessage();
         } else {
             status = HttpResponseStatus.INTERNAL_SERVER_ERROR;
-            LOGGER.error("unhandled exception:", ex);
+            LOGGER.debug("unhandled exception:", ex);
```

Comment:
We should send back a "you sent me something invalid" in response to the request, but I'd propose to keep it out of the log.